src/_includes/head.html

```diff
@@ -1,24 +1,43 @@
-<title>{% if page.title %}{{ page.title }}{% else %}Software Craftsmanship and Agile Development{% endif %} | Codurance | Craft at Heart | London | Barcelona</title>
+<title>
+    {% if page.url == "/software-modernisation/" and site.lang == "es" %}
+        {{ page.title-es }}
+    {% elsif page.url == "/software-modernisation/" and site.lang != "es" %}
+        {{ page.title }}
+    {% else %}
+        {% if page.title %}
+            {{ page.title }}
+        {% else %}
+            Software Craftsmanship and Agile Development
+        {% endif %}
+        | Codurance | Craft at Heart | London | Barcelona
+    {% endif %}
+</title>
+
```

Comment:
This is starting to get messy! It's still readable just about but shouldn't this logic be in a single place wherever `page.title` is defined. 

This is not a comment to be addressed before merging (I know you guys are working super fast for some reasons.)  But outside of this PR would should talk about the website

Reply:
Yes the meta on the site needs a complete review and this is by no means how I would like it to be. But to get this out I made these changes for now. I would want the logic to calculate the right text for the title's, descriptions etc. to sit in a separate file and just pull in the result into a variable for each meta tag value.
The page title tag could be all on one line (which it was to start with) but I left with a separate part of the if on each line just for readability of the if statements.

Reply:
Cool. We're in agreement